src/test/java/com/cloudbees/jenkins/plugins/docker_build_env/FunctionalTests.java

```diff
@@ -47,12 +47,12 @@ public void run_inside_pulled_container() throws Exception {
     @Test
     public void run_inside_built_container() throws Exception {
         FreeStyleProject project = jenkins.createFreeStyleProject();
-        project.setScm(new SingleFileSCM("Dockerfile", IOUtils.toString(getClass().getResourceAsStream("/Dockerfile"), "UTF-8")));
+        project.setScm(new SingleFileSCM("Dockerfile", "FROM ubuntu:14.04"));
 
         project.getBuildWrappersList().add(
                 new DockerBuildWrapper(
-                        new DockerfileImageSelector(".", "$WORKSPACE/Dockerfile"),
-                        "", new DockerServerEndpoint("", ""), "", true, false, Collections.<Volume>emptyList(), null, "cat", false, "bridge", null, null)
+                        new DockerfileImageSelector(".", "Dockerfile"),
```

Comment:
Seems like it could stil have $WORKSPACE... unless it needs to be changed?

Reply:
See comment above. I agree, $WORKSPACE seems reasonable here. This was the only way I could get the test running and passing. If it's an issue, I can try to work out the details of @LocalData. Any hints are welcome. More info here: https://wiki.jenkins.io/display/JENKINS/Unit+Test#UnitTest-@LocalData